src/Microsoft.AspNetCore.Authentication.JwtBearer/JwtBearerHandler.cs

```diff
@@ -185,11 +185,20 @@ internal class JwtBearerHandler : AuthenticationHandler<JwtBearerOptions>
 
         protected override async Task<bool> HandleUnauthorizedAsync(ChallengeContext context)
         {
-            var authResult = await HandleAuthenticateOnceAsync();
+            Exception failure = null;
```

Comment:
Tried to push up the the exception catching: ([see here](https://github.com/aspnet/Security/tree/troy-927-2)) it will break some user's scenario. Because exception which used to create the middleware ([here](https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNetCore.Authentication/AuthenticationMiddleware.cs#L68)) is now safely captured. Exception is not thrown from middleware any more. This will break all the customer scenarios where leverage exception to break through authenticate middleware (it's not a right thing to do though).